docs/GHAFileExplained.md

```diff
@@ -50,30 +50,33 @@ jobs:
     # Determines build matrix. This is a list of PHP versions, databases and
     # branches to test our project against. For each combination a separate
     # build will be created. For example below 6 builds will be created in
-    # total (7.2-pgsql, 7.2-mariadb, 7.3-pgsql, 7.3-mariadb, etc.). If we add
+    # total (7.3-pgsql, 7.3-mariadb, 7.4-pgsql, 7.4-mariadb, etc.). If we add
     # another branch, total number of builds will become 12.
     # If you need to use PHP 7.0 and run phpunit coverage test, make sure you are
     # using ubuntu-16.04 virtual environment in this case to have phpdbg or
     # this version in the system.
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.2', '7.3', '7.4']
-        moodle-branch: ['MOODLE_310_STABLE']
+        php: ['7.3', '7.4', '8.0']
```

Comment:
I suggest to add same changes (version and branch matrices) to `gha.dist.yml`, so that it looks complete for the php8 test scenario.

Reply:
New mini-commit (https://github.com/moodlehq/moodle-plugin-ci/pull/102/commits/bfe3366933791c49bebcb26a89b6dbe93653d9ec) makes the dist files to match the Explained docs.